### DISCUSSION TITLE
can't find lib32gcc1 on pop_os 20.10

### DISCUSSION THREAD

**Discussion by deadclam** [GameServerManagers/LinuxGSM#3279] [Q&A] [answered]
Running Pop!_os 20.10 and getting the error about not having lig32gcc1.  I've successfully installed lib32gcc-s1, and have installed SteamCMD and run it to get the server I'm trying to host.  I installed all the other missing dependencies manually without issue.

Each time I do start, monitor etc it does the check and errors out.  It looks like the start works at first as running details directly after shows the status as ONLINE, but then it goes back offline right after.

Something else I should be doing?

running LinuxGSM version 21.1.3

Distro Details
=================================
Distro:    Pop!_OS 20.10
Arch:      x86_64
Kernel:    5.8.0-7642-generic
Hostname:  pop-os
Uptime:    1d, 15h, 18m
tmux:      tmux 3.1b
glibc:     2.32

**Comment by rconjoe** [comment, marked as answer]
There is an outstanding PR awaiting approval that should solve this during LinuxGSM's dependency check: #3274

**Comment by deadclam** [comment]
thanks for the update @rconjoe .  I'll keep an eye out for the new version.
